lib/truncate.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const MIN_LENGTH = 1021
```

Comment:
Seems like a MAX length not a MIN length?
```suggestion
const MAX_LENGTH = 1021
```